src/main.py

```diff
@@ -8,12 +8,16 @@
 import threading
 from sbio.dimmer import Dimmer
 from sbio.pushbutton import PushButton
+from api.weather.ecWeather import ecWxWorker
+from api.weather.owmWeather import owmWxWorker
+from api.weather.ecAlerts import ecWxAlerts
+from api.weather.nwsAlerts import nwsWxAlerts
 from renderer.matrix import Matrix
 import debug
 
 SCRIPT_NAME = "NHL-LED-SCOREBOARD"
 
-SCRIPT_VERSION = "1.1.3"
+SCRIPT_VERSION = "1.2.0 - Wx version"
```

Comment:
This was just my holder value for the version, you can decide what the version should be.